api/v1alpha1/stage_types.go

```diff
@@ -611,6 +618,24 @@ type FreightHistoryEntry struct {
 	// Freight is a map of FreightReference objects, indexed by their Warehouse
 	// origin.
 	Freight map[string]FreightReference `json:"items,omitempty" protobuf:"bytes,1,rep,name=items" protobuf_key:"bytes,1,opt,name=key" protobuf_val:"bytes,2,opt,name=value"`
+	// VerificationHistory is a stack of recent VerificationInfo. By default,
+	// the last ten VerificationInfo are stored.
+	VerificationHistory VerificationInfoStack `json:"verificationHistory,omitempty" protobuf:"bytes,2,rep,name=verificationHistory"`
+}
+
+// FreightNames returns a sorted and comma-delimited list of the names of the
+// Freight in the FreightHistoryEntry. This is useful for when an identifier is
+// needed to represent a precise combination of Freight.
+func (f *FreightHistoryEntry) FreightNames() string {
```

Comment:
Wonder if it may be slightly better for the caller to decide if `strings.Join` should be used, and have this return `[]string`.

Alternatively, a separate method could be added which calls `FreightNames` to present a string identifier then.

Reply:
Yeah... reviewing how widely this _isn't_ used, this could really just be a private func closer to the verification logic.

Will move.